backend/api_calls.py

```diff
@@ -358,17 +359,17 @@ def AddPlayer(request, firebase):
   game = request['gameId']
   player = request['playerId']
   user = request['userId']
-  start_as_zombie = request['startAsZombie']
 
   player_info = {'gameId': game}
   results.append(firebase.put('/users/%s/players' % user, player, player_info))
 
   player_info = {
     'gameId': game,
     'userId': user,
-    'canInfect': start_as_zombie,
+    'canInfect': (request['startAsZombie'] or request['beSecretZombie']),
     'needGun' : request['needGun'],
-    'startAsZombie' : start_as_zombie,
+    'gotEquipment' : request['gotEquipment'],
```

Comment:
I'm trying to make all args required as much as possible. It's the client code filling them in so it's easier to just set them vs handling maybe-setting them and it's easier on the server vs dealing with missing keys or determining a default.